rosidl_typesupport_c/include/rosidl_typesupport_c/type_support_map.h

```diff
@@ -22,15 +22,26 @@ extern "C"
 {
 #endif
 
+/// Standard structure for encapsulating information about a rosidl interface type's support.
 typedef struct type_support_map_t
 {
   // TODO(dirk-thomas) const should not be defined for the fields
   // but should be set for the struct when it is being used
   // same for rosidl_message_type_support_t et al
+
+  /// Number of type_supports in the data array.
```

Comment:
It would be good to note that the other arrays must have the length as specified by `size`.

Reply:
Fixed here and in _cpp